### PR TITLE
OdometryData

### PR DESCRIPTION
**Pull request by randaz81**
Added data type OdometryData, generated by `yarp\src\libYARP_dev\src\idl\OdometryData.thrift`

### CODE REVIEW

**Comment by randaz81** [comment]
@drdanz I noticed that the comments I put in the .thrift file are removed in the generated code. Am I missing something or the comments are just not implemented?

**Comment by randaz81** [comment]
I also found an open issue related to thrift comments: https://github.com/robotology/yarp/issues/998

**Comment by drdanz** [comment]
I think have to use the doxygen comment syntax `/**... */` to get them in the generated code

**Comment by randaz81** [comment]
> I think have to use the doxygen comment syntax `/**... */` to get them in the generated code

It works perfectly 👍

File: src/libYARP_dev/src/CMakeLists.txt [hunk 1]
```diff
@@ -212,8 +212,10 @@ endif()
 include(YarpIDL)
 
 set(YARP_dev_IDL idl/stateExt.thrift
-                 idl/audioBufferSizeData.thrift)
-				 
+                 idl/audioBufferSizeData.thrift
+                 idl/OdometryData.thrift
+				 idl/OdometryData6D.thrift)
```
Comment (drdanz) [inline src/libYARP_dev/src/CMakeLists.txt]: Please fix this alignment

**Pull request merged**